metabase/metabase
# fix x-rays visible when disabled

Closes https://github.com/metabase/metabase/issues/26571

### Description

When x-rays disabled in settings we should not render them even for admins on the home page.

### How to verify

1. Run a completely new instance
2. Go to Admin Settings -> General -> Disable x-rays
3. Go to the home page and ensure there is no x-rays

### Checklist
